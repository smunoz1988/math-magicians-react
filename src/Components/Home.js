const Home = () => (

  <div className="homeContainer">

    <h1>
      Welcome to our page!
    </h1>

    <p>
      Lorem ipsum dolor sit, amet consectetur adipisicing elit.
      Repellat repudiandae dolor similique,
      nulla quam exercitationem,eveniet rem expedita aliquid excepturi
      dicta aspernatur. Tempora dolorem consequuntur vero minima maiores dicta at?.
      Lorem ipsum dolor sit, amet consectetur adipisicing elit.
      Repellat repudiandae dolor similique, nulla quam exercitationem,
      eveniet rem expedita aliquid excepturi dicta aspernatur.
      Tempora dolorem consequuntur vero minima maiores dicta at?
    </p>

    <p>
      Lorem ipsum dolor sit, amet consectetur adipisicing elit.
      Repellat repudiandae dolor similique, nulla quam exercitationem,
      eveniet rem expedita aliquid excepturi dicta aspernatur. Tempora dolorem
      consequuntur vero minima maiores dicta at?.
      Lorem ipsum dolor sit, amet consectetur adipisicing elit.
      Repellat repudiandae dolor similique, nulla quam exercitationem,
      eveniet rem expedita aliquid excepturi dicta aspernatur.
      Tempora dolorem consequuntur vero minima maiores dicta at?
    </p>

  </div>

);

export default Home;
