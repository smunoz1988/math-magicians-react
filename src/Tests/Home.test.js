import { render, screen } from '@testing-library/react';
import Home from '../components/home.js';

describe('it should show content when the user enter the Math home page', () => {
  it('displays title', () => {
    const { container } = render(<Home />);

    expect(screen.getByText('Welcome to our page!')).toBeTruthy();
    expect(container.firstChild).toMatchSnapshot();
  });

  it('when home page is render, the first paragraph should be display', () => {
    const { container } = render(<Home />);
    const paragraph = 'Lorem ipsum dolor sit, amet consectetur adipisicing elit. Repellat repudiandae dolor similique, nulla quam exercitationem,eveniet rem expedita aliquid excepturi dicta aspernatur. Tempora dolorem consequuntur vero minima maiores dicta at?. Lorem ipsum dolor sit, amet consectetur adipisicing elit. Repellat repudiandae dolor similique, nulla quam exercitationem, eveniet rem expedita aliquid excepturi dicta aspernatur. Tempora dolorem consequuntur vero minima maiores dicta at?';

    expect(screen.getByText(paragraph)).toBeTruthy();
    expect(container.firstChild).toMatchSnapshot();
  });
});
