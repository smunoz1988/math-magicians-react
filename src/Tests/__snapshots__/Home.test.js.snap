// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it should show content when the user enter the Math home page displays title 1`] = `
<div
  class="homeContainer"
>
  <h1>
    Welcome to our page!
  </h1>
  <p>
    Lorem ipsum dolor sit, amet consectetur adipisicing elit. Repellat repudiandae dolor similique, nulla quam exercitationem,eveniet rem expedita aliquid excepturi dicta aspernatur. Tempora dolorem consequuntur vero minima maiores dicta at?. Lorem ipsum dolor sit, amet consectetur adipisicing elit. Repellat repudiandae dolor similique, nulla quam exercitationem, eveniet rem expedita aliquid excepturi dicta aspernatur. Tempora dolorem consequuntur vero minima maiores dicta at?
  </p>
  <p>
    Lorem ipsum dolor sit, amet consectetur adipisicing elit. Repellat repudiandae dolor similique, nulla quam exercitationem, eveniet rem expedita aliquid excepturi dicta aspernatur. Tempora dolorem consequuntur vero minima maiores dicta at?. Lorem ipsum dolor sit, amet consectetur adipisicing elit. Repellat repudiandae dolor similique, nulla quam exercitationem, eveniet rem expedita aliquid excepturi dicta aspernatur. Tempora dolorem consequuntur vero minima maiores dicta at?
  </p>
  <img
    alt="logo"
    class="homeImageCont"
    src="970426.jpg"
  />
</div>
`;

exports[`it should show content when the user enter the Math home page when home page is render, the first paragraph should be display 1`] = `
<div
  class="homeContainer"
>
  <h1>
    Welcome to our page!
  </h1>
  <p>
    Lorem ipsum dolor sit, amet consectetur adipisicing elit. Repellat repudiandae dolor similique, nulla quam exercitationem,eveniet rem expedita aliquid excepturi dicta aspernatur. Tempora dolorem consequuntur vero minima maiores dicta at?. Lorem ipsum dolor sit, amet consectetur adipisicing elit. Repellat repudiandae dolor similique, nulla quam exercitationem, eveniet rem expedita aliquid excepturi dicta aspernatur. Tempora dolorem consequuntur vero minima maiores dicta at?
  </p>
  <p>
    Lorem ipsum dolor sit, amet consectetur adipisicing elit. Repellat repudiandae dolor similique, nulla quam exercitationem, eveniet rem expedita aliquid excepturi dicta aspernatur. Tempora dolorem consequuntur vero minima maiores dicta at?. Lorem ipsum dolor sit, amet consectetur adipisicing elit. Repellat repudiandae dolor similique, nulla quam exercitationem, eveniet rem expedita aliquid excepturi dicta aspernatur. Tempora dolorem consequuntur vero minima maiores dicta at?
  </p>
  <img
    alt="logo"
    class="homeImageCont"
    src="970426.jpg"
  />
</div>
`;
